### ISSUE TITLE
CallbackRegistryMixin:UnregisterEvents() calls wrong function if event table is passed

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#105]
The [CallbackRegistryMixin:UnregisterEvents](https://github.com/Gethe/wow-ui-source/blob/5669806c19d84e0da9f324f665855b30730f0457/SharedXML/CallbackRegistry.lua#L84)() function if supplied a table of event names will call the [UnregisterCallback](https://github.com/Gethe/wow-ui-source/blob/5669806c19d84e0da9f324f665855b30730f0457/SharedXML/CallbackRegistry.lua#L72)() method with a nil `owner` parameter and trigger an error condition. I believe this is meant to actually be calling [UnregisterAllCallbacksByEvent](https://github.com/Gethe/wow-ui-source/blob/5669806c19d84e0da9f324f665855b30730f0457/SharedXML/CallbackRegistry.lua#L76)() instead.

```lua
local CBR = CreateFromMixins(CallbackRegistryMixin)
CBR:OnLoad();
CBR:GenerateCallbackEvents({ "OnMeow" });
CBR:UnregisterEvents(CBR.Event);  -- error: "CallbackRegistryMixin:UnregisterCallback owner is nil."
```

**Comment by Meorawr** [comment]
For TBC this is resolved in 2.5.1.38453.

**Comment by Meorawr** [comment]
Should be fixed in 9.1.0.38511.
